src/tests/functional/frame_tests.ts

```diff
@@ -229,7 +232,7 @@ test("test the turbo:frame-missing event following a link to a page without a ma
   await nextEventNamed(page, "turbo:load")
 
   assert.equal(pathname(page.url()), "/src/tests/fixtures/frames.html")
-  assert.equal(await innerHTMLForSelector(page, "#missing"), "")
+  assert.ok(await hasSelector(page, "#missing-frame-link"))
```

Comment:
I _think_ the original assertion wasn't right here. When navigating back to the previous page, we'd expect the original content of that frame to be there. So I changed this one to check for the presence of the link that was on the page before.